build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+	java
```

Comment:
> The Application plugin facilitates creating an executable JVM application. It makes it easy to start the application locally during development, and to package the application as a TAR and/or ZIP including operating system specific start scripts.
[The Application Plugin ](https://docs.gradle.org/current/userguide/application_plugin.html)

What about using the application plugin instead. Given that we need to execute that JAR and it'll make easier to run it

Reply:
done